realm/src/main/java/io/realm/Realm.java

```diff
@@ -1778,6 +1789,25 @@ public String getPath() {
         return path;
     }
 
+
+    /**
+     * Override the standard behavior of all classes extended RealmObject being part of the schema.
+     * Use this method to define the schema as only the classes given here.
+     *
+     * This class must be called before calling {@link #getInstance(android.content.Context)}
+     *
+     * If {@code null} is given as parameter, the Schema is reset to use all known classes.
+     *
+     */
+    static void setSchema(Class<? extends RealmObject>... schemaClass) {
```

Comment:
This method is package private, so only used for testing. The assumption was that the most important thing is to quickly add a small number of classes with less code.
```
Realm.setSchema(Foo.class, Bar.class); // reads better than
Realm.setSchema(Arrays.asList(Foo.class, Bar.class)); 
```